pkg/cmd/label/clone_test.go

```diff
@@ -392,59 +392,87 @@ func TestCloneRun(t *testing.T) {
 					),
 				)
 			},
-			wantErr: true,
+			wantErr:    true,
+			wantErrMsg: "HTTP 422: Validation Failed (https://api.github.com/repos/OWNER/REPO/labels)\nLabel.color is invalid",
 		},
 		{
 			name: "clones pages of labels",
 			tty:  true,
 			opts: &cloneOptions{SourceRepo: ghrepo.New("cli", "cli"), Force: true},
 			httpStubs: func(reg *httpmock.Registry) {
-				for i := 0; i < pages; i++ {
-					labels := make([]label, listLimit)
-					for j := range labels {
-						labels[i] = label{
-							Name:  fmt.Sprintf("label%d", i*listLimit+j),
-							Color: "00ff00",
+				reg.Register(
+					httpmock.GraphQL(`query LabelList\b`),
+					httpmock.StringResponse(`
```

Comment:
@samcoe any reason we wouldn't want to also verify the variable inputs? This decreases coverage by ignoring request assertions.

Reply:
No, just missed it when making changes. Thanks for pointing it out. It should be fixed now.